blivet/blivet.py

```diff
@@ -46,7 +46,7 @@
 from .flags import flags
 from .platform import platform as _platform
 from .formats import get_format
-from .osinstall import FSSet, find_existing_installations
+from .osinstall import FSSet, StorageDiscoveryConfig find_existing_installations
```

Comment:
All right, well I fixed it in my latest push (which rebased to latest 3.0-devel), even though it is innocuous. ;)